sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -611,6 +619,37 @@ var sloBurnRateTriggerConditionSchema = map[string]*schema.Schema{
 	}, sloBurnRateConditionCriticalOrWarningAtleastOneKeys),
 }
 
+var logsAnomalyTriggerConditionSchema = map[string]*schema.Schema{
+	"field": {
+		Type:     schema.TypeString,
+		Required: true,
+	},
+	"direction": {
+		Type:         schema.TypeString,
+		Optional:     true,
+		Default:      "Both",
+		ValidateFunc: validation.StringInSlice([]string{"Both", "Up", "Down"}, false),
+	},
+	"anomaly_detector_type": {
+		Type:         schema.TypeString,
+		Required:     true,
+		ValidateFunc: validation.StringInSlice([]string{"Cluster"}, false),
+	},
+	"sensitivity": {
+		Type:     schema.TypeFloat,
+		Optional: true,
+		Default:  0.5,
+	},
+	"critical": nested(false, schemaMap{
+		"min_anomaly_count": {
+			Type:     schema.TypeInt,
+			Optional: true,
+			Default:  1,
```

Comment:
adding validation for this would be hard. so we can ignore adding one to this.

Reply:
The API has the same default value. Even if I remove it from here, we will still use 1 as default value.